### DISCUSSION TITLE
Request: Create split view tab for PiP players

### DISCUSSION THREAD

**Discussion by Dean-Corso** [zen-browser/desktop#1786] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

I like the split view feature to display more than one website into one single tab. So I would like to have similar feature just for the PiP video player showing just the players only on them. Lets say you visit a website and wanna watch few live streams or videos at once and having them bundled on one tab (split view) not outside of browser window (Picture-in-Picture: auto-open on tab switch). Its like the split view you have right now for entire websites but in this case I want to have it just for the players itself without the website around. I think its a good idea. There are similar tools on GH like [Gridplayer](https://github.com/vzhd1701/gridplayer) what can do that and something like that having directly in browser would be great.

By the way, in normal split view mode there is a problem with the full screen mode of those media / video players. If you have more than one website into split view (few YT videos / players) and you press on Full screen button on the video player then does just use the split area as screen area instead using entire screen itself like it does without split view. That's a disadvantage.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.5 (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1761/request-create-split-view-tab-for-pip-players">ZEN-1761 Request: Create split view tab for PiP players</a></p>
